### PR TITLE
ci: use riot for psycopg and sqlalchemy

### PR DESCRIPTION
**Pull request by majorgreys**
## Description

Use riot for psycopg (adding the 6 tests for latest to the test matrix) and sqlalchemy

```
$ tox -l | grep psycopg_contrib
psycopg_contrib-py27-psycopg224
psycopg_contrib-py27-psycopg225
psycopg_contrib-py27-psycopg226
psycopg_contrib-py27-psycopg227
psycopg_contrib-py27-psycopg228
psycopg_contrib-py35-psycopg224
psycopg_contrib-py35-psycopg225
psycopg_contrib-py35-psycopg226
psycopg_contrib-py35-psycopg227
psycopg_contrib-py35-psycopg228
psycopg_contrib-py36-psycopg224
psycopg_contrib-py36-psycopg225
psycopg_contrib-py36-psycopg226
psycopg_contrib-py36-psycopg227
psycopg_contrib-py36-psycopg228
psycopg_contrib-py37-psycopg227
psycopg_contrib-py37-psycopg228
psycopg_contrib-py38-psycopg228
psycopg_contrib-py39-psycopg228
$ tox -l | grep psycopg_contrib | wc -l
19
$ tox -l | grep sqlalchemy_contrib
sqlalchemy_contrib-py27-sqlalchemy10-psycopg228-mysqlconnector
sqlalchemy_contrib-py27-sqlalchemy11-psycopg228-mysqlconnector
sqlalchemy_contrib-py27-sqlalchemy12-psycopg228-mysqlconnector
sqlalchemy_contrib-py27-sqlalchemy13-psycopg228-mysqlconnector
sqlalchemy_contrib-py27-sqlalchemy-psycopg228-mysqlconnector
sqlalchemy_contrib-py35-sqlalchemy10-psycopg228-mysqlconnector
sqlalchemy_contrib-py35-sqlalchemy11-psycopg228-mysqlconnector
sqlalchemy_contrib-py35-sqlalchemy12-psycopg228-mysqlconnector
sqlalchemy_contrib-py35-sqlalchemy13-psycopg228-mysqlconnector
sqlalchemy_contrib-py35-sqlalchemy-psycopg228-mysqlconnector
sqlalchemy_contrib-py36-sqlalchemy10-psycopg228-mysqlconnector
sqlalchemy_contrib-py36-sqlalchemy11-psycopg228-mysqlconnector
sqlalchemy_contrib-py36-sqlalchemy12-psycopg228-mysqlconnector
sqlalchemy_contrib-py36-sqlalchemy13-psycopg228-mysqlconnector
sqlalchemy_contrib-py36-sqlalchemy-psycopg228-mysqlconnector
sqlalchemy_contrib-py37-sqlalchemy10-psycopg228-mysqlconnector
sqlalchemy_contrib-py37-sqlalchemy11-psycopg228-mysqlconnector
sqlalchemy_contrib-py37-sqlalchemy12-psycopg228-mysqlconnector
sqlalchemy_contrib-py37-sqlalchemy13-psycopg228-mysqlconnector
sqlalchemy_contrib-py37-sqlalchemy-psycopg228-mysqlconnector
sqlalchemy_contrib-py38-sqlalchemy10-psycopg228-mysqlconnector
sqlalchemy_contrib-py38-sqlalchemy11-psycopg228-mysqlconnector
sqlalchemy_contrib-py38-sqlalchemy12-psycopg228-mysqlconnector
sqlalchemy_contrib-py38-sqlalchemy13-psycopg228-mysqlconnector
sqlalchemy_contrib-py38-sqlalchemy-psycopg228-mysqlconnector
sqlalchemy_contrib-py39-sqlalchemy10-psycopg228-mysqlconnector
sqlalchemy_contrib-py39-sqlalchemy11-psycopg228-mysqlconnector
sqlalchemy_contrib-py39-sqlalchemy12-psycopg228-mysqlconnector
sqlalchemy_contrib-py39-sqlalchemy13-psycopg228-mysqlconnector
sqlalchemy_contrib-py39-sqlalchemy-psycopg228-mysqlconnector
$ tox -l | grep sqlalchemy_contrib | wc -l
30
```

```
$ riot list | grep "^psycopg"
psycopg  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.4.0'
psycopg  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.5.0'
psycopg  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.6.0'
psycopg  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.7.0'
psycopg  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.8.0'
psycopg  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2'
psycopg  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.4.0'
psycopg  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.5.0'
psycopg  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.6.0'
psycopg  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.7.0'
psycopg  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.8.0'
psycopg  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2'
psycopg  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.4.0'
psycopg  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.5.0'
psycopg  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.6.0'
psycopg  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.7.0'
psycopg  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.8.0'
psycopg  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2'
psycopg  Interpreter(_hint='3.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.7.0'
psycopg  Interpreter(_hint='3.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.8.0'
psycopg  Interpreter(_hint='3.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2'
psycopg  Interpreter(_hint='3.8') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.8.0'
psycopg  Interpreter(_hint='3.8') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2'
psycopg  Interpreter(_hint='3.9') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2~=2.8.0'
psycopg  Interpreter(_hint='3.9') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'psycopg2'
$ riot list | grep "^psycopg" | wc -l
25
$ riot list | grep "^sqlalchemy"
sqlalchemy  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.0.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.1.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.2.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.3.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='2.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.0.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.1.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.2.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.3.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.5') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.0.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.1.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.2.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.3.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.6') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.0.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.1.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.2.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.3.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.7') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.8') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.0.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.8') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.1.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.8') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.2.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.8') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.3.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.8') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.9') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.0.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.9') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.1.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.9') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.2.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.9') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy~=1.3.0' 'psycopg2~=2.8.0' 'mysql-connector-python'
sqlalchemy  Interpreter(_hint='3.9') 'mock' 'pytest' 'coverage' 'pytest-cov' 'opentracing' 'sqlalchemy' 'psycopg2~=2.8.0' 'mysql-connector-python'
$ riot list | grep "^sqlalchemy" | wc -l
30
```

### CODE REVIEW

File: riotfile.py [hunk 1]
```diff
@@ -298,6 +322,19 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
...
         ),
+        Venv(
+            name="sqlalchemy",
+            command="pytest {cmdargs} tests/contrib/sqlalchemy",
+            venvs = [
+                Venv(
+                    pys=select_pys(),
+                    pkgs={
+                        "sqlalchemy": ["~=1.0.0", "~=1.1.0", "~=1.2.0", "~=1.3.0", latest],
+                        "psycopg2": ["~=2.8.0"],
```
Comment (jd) [inline riotfile.py:333]: I feel like we should install `SQLAlchemy[postgresql]` and not psycopg2 directly here.
That should be changed in a different PR anyway.

**Review by jd — APPROVED** [review]

File: riotfile.py [hunk 2]
```diff
@@ -298,6 +322,20 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
...
+                        "mysql-connector-python": latest,
```
Comment (majorgreys) [inline riotfile.py:334]: Missed this earlier

Comment (jd) [reply riotfile.py:334]: ditto :)

**Review by jd — APPROVED** [review]
